storybook/cypress/e2e/viewer-grid.cy.ts

```diff
@@ -1,12 +1,24 @@
 const url = "http://localhost:45000/iframe.html?id=core-viewer-grid--viewer-grid";
 const urlOverlayClosed = `${url}&args=overlayOpen:false`;
 const urlOverlayOpened = `${url}&args=overlayOpen:true`;
-const urlNoOverlay = `${urlOverlayOpened}&args=noOverlay:true`;
```

Comment:
Dit heb ik weggehaald, want was iets ouds, of in ieder geval  `noOverlay:true` deed niets

Reply:
Goed gevonden